plugins/Widgetize/lang/en.json

```diff
@@ -2,7 +2,8 @@
     "Widgetize": {
         "OpenInNewWindow": "Open in a new window",
         "PluginDescription": "Display any Matomo report in your website or app with a simple Embed HTML tag.",
-        "ViewAccessRequired": "This user has at least some write access. Only tokens of users who have only view access can be used.",
-        "TopLinkTooltip": "Export Matomo Reports as Widgets and embed the Dashboard in your app as an iframe."
+        "ViewAccessRequired": "This user has at least some write access. Only tokens of users who have only view access can be used. See %1$s for more information.",
+        "TopLinkTooltip": "Export Matomo Reports as Widgets and embed the Dashboard in your app as an iframe.",
+        "TooHighAccessLevel": "This user has too much access for embedding widgets. Please use a user less access."
```

Comment:
Maybe we could say `This user has super user access. For embedding widgets please use the token of a user that has view access only.`? could then also maybe link to https://matomo.org/faq/troubleshooting/faq_147/ if needed?

Reply:
This error could also occur if the allow admin/write token auth setting is set, so will change it to`This user has super user access. For embedding widgets super user token auths are not allowed. %1$sSee our faq for more information.%2$s`.